p3/forms.py

```diff
@@ -124,14 +124,15 @@ def save(self, *args, **kwargs):
 
         return talk
 
+
 class P3SubmissionAdditionalForm(P3TalkFormMixin, cforms.TalkForm):
-    duration = P3SubmissionForm.base_fields['duration']
+    duration         = P3SubmissionForm.base_fields['duration']
```

Comment:
Thanks, yes I will PEP-8 it.

I agree on the: ‘I’m not going to change it all over the project'

Alexandre Manhães Savio  
PhD, Medical Imaging, Machine Learning
Grupo de Inteligencia Computacional UPV/EHU
home | email: alexsavio@gmail.com

From: fpliger <notifications@github.com>
Reply: EuroPython/epcon <reply@reply.github.com>>
Date: 14 Feb 2015 at 09:50:09
To: EuroPython/epcon <epcon@noreply.github.com>>
Cc: Alexandre M. S. <alexsavio@gmail.com>>
Subject:  Re: [epcon] Fix p3/templates not loading. (#8)  

In p3/forms.py:

>  class P3SubmissionAdditionalForm(P3TalkFormMixin, cforms.TalkForm):
> -    duration = P3SubmissionForm.base_fields['duration']
> +    duration         = P3SubmissionForm.base_fields['duration']
IMHO we should all try to be PEP8-ish compliant.... I'm -0 on this style choice, as I don't think you'll be changing it all over the project so mixing up styles like this is not a great choice

—
Reply to this email directly or view it on GitHub.